packages/orbit-components/src/ListChoice/ListChoice.stories.tsx

```diff
@@ -7,239 +9,185 @@ import RenderInRtl from "../utils/rtl/RenderInRtl";
 
 import ListChoice from ".";
 
+Button.displayName = "Button";
+ListChoice.displayName = "ListChoice";
+
 const getIcon = (source: string | null) => source && Icons[source];
 
-export default {
+const meta: Meta<typeof ListChoice> = {
   title: "ListChoice",
-};
-
-export const Default = ({ title, description, icon }) => {
-  const Icon = getIcon(icon);
-
-  return (
-    <ListChoice
-      title={title}
-      description={description}
-      icon={Icon && <Icon />}
-      onClick={action("onClick")}
-    />
-  );
-};
+  component: ListChoice,
 
-Default.story = {
   parameters: {
-    info: "Some description about this type of ListChoice in general.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
+    controls: {
+      exclude: ["onClick"],
+    },
   },
-};
 
-Default.args = {
-  title: "Choice Title",
-  description: "Further description",
-  icon: "Accommodation",
-};
+  args: {
+    title: "Choice Title",
+    description: "Further description",
+    icon: "Accommodation",
+    selectable: true,
+    selected: false,
+    disabled: false,
+    tabIndex: 0,
+    onClick: action("onClick"),
+    action: null,
```

Comment:
You set `action` here as null, but then in `WithAction` story you set it as undefined and you also exclude it in every story. Can't we just not set it at all?

Reply:
GP, let's move it out.